lite/lite_server.py

```diff
@@ -13,14 +13,19 @@
 import json
 import urllib
 import gzip
+import zipfile
+
+from multipart import parse_options_header, MultipartParser
 
 PORT = 5808
 ROOT = os.path.abspath("static")
 IS_SYSTEMCORE = os.uname().nodename == "robot"
 EXTRA_ASSETS_PATH = "/home/systemcore/ascope_assets" if IS_SYSTEMCORE else os.path.abspath("ascope_assets")
 BUNDLED_ASSETS_PATH = os.path.join(ROOT, "bundledAssets")
 ALLOWED_LOG_SUFFIXES = [".wpilog", ".rlog"]  # Hoot not supported
-ENABLE_LOG_DOWNLOADS = IS_SYSTEMCORE or "--enable-logs" in sys.argv
+# todo: is backwards compatibility important here?
```

Comment:
I think it's OK to remove the `--enable-logs` version of the argument here, since there aren't any scripts that need to use this argument.